src/Catalog/CollectorBase.cs

```diff
@@ -12,17 +12,14 @@ namespace NuGet.Services.Metadata.Catalog
 {
     public abstract class CollectorBase
     {
+        private readonly ITelemetryService _telemetryService;
```

Comment:
>private [](start = 8, length = 7)

Talked to Joel offline, this will become protected later.

Reply:
Basically, this needs to be protected to be useful. In a follow-up PR I have done this.

Reply:
Fixed.